guides/source/active_record_migrations.md

```diff
@@ -482,34 +604,37 @@ change_column :products, :part_number, :text
 
 This changes the column `part_number` on products table to be a `:text` field.
 
-NOTE: The `change_column` command is **irreversible**.
-You should provide your own `reversible` migration, like we discussed
-[before](#making-the-irreversible-possible).
+NOTE: The `change_column` command is **irreversible**. You should provide your
```

Comment:
What does it mean 'You should provide'...does that mean it is always better not to use 'change_column' or does it mean that it doesn't change in a backwards direction?